tools/docker/builder/openwrt/build.sh

```diff
@@ -56,7 +57,19 @@ main() {
     while true; do
         case "$1" in
             -h | --help)               usage; exit 0; shift ;;
-            -d | --target-device)      TARGET_DEVICE="$2"; shift ; shift ;;
+            -d | --target-device)
+		TARGET_DEVICE="$2"; shift ; shift
+		case "$TARGET_DEVICE" in
+		    turris-omnia)
+			TARGET=mvebu
+			;;
+		    *)
+			err "Unknown target device: $TARGET_DEVICE"
+			info "Currently supported targets are: turris-omnia"
+			;;
+		esac
+		;;
+	    -f | --prpl-feed)          PRPL_FEED="$2"; shift; shift ;;
```

Comment:
missing a space..

Reply:
Crap, tabs vs spaces again. Let me untabify it.